### PR TITLE
Install MongoDB

### PR DESCRIPTION
**Pull request by derjubu**
I installed mongoDB for this project and created a server route to save the destination added.
For the app to work, I had to keep the functionality to save the data in local storage. This will be removed in a later PR.

### CODE REVIEW

File: src/app/pages/DestinationDetailView/DestinationDetailView.tsx [hunk 1]
```diff
@@ -9,6 +9,22 @@ export default function DestinationDetailView(): JSX.Element {
     window.localStorage.getItem('destination') || '[]'
   );
 
+  async function saveDestination(newDestination: DestinationType) {
+    console.log('click');
```
Comment (janhendrikstruve) [inline src/app/pages/DestinationDetailView/DestinationDetailView.tsx]: Console.log 🔍

File: src/app/pages/DestinationDetailView/DestinationDetailView.tsx [hunk 2]
```diff
@@ -9,6 +9,22 @@ export default function DestinationDetailView(): JSX.Element {
...
+    const response = await fetch('/api/location/', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ newDestination }),
+    });
+    if (response.status === 200) {
+      console.log('Done!');
```
Comment (janhendrikstruve) [inline src/app/pages/DestinationDetailView/DestinationDetailView.tsx]: Console.log 🤠

File: src/app/pages/DestinationDetailView/DestinationDetailView.tsx [hunk 3]
```diff
@@ -9,6 +9,22 @@ export default function DestinationDetailView(): JSX.Element {
...
+    } else {
+      console.log('Fehler');
```
Comment (janhendrikstruve) [inline src/app/pages/DestinationDetailView/DestinationDetailView.tsx]: Console.log 🧐

File: src/app/pages/DestinationDetailView/DestinationDetailView.tsx (hunk 1 above)
Comment (lajanzen) [inline src/app/pages/DestinationDetailView/DestinationDetailView.tsx]:
```suggestion
```

File: src/server.ts [hunk 4]
```diff
@@ -20,6 +35,8 @@ app.get('*', (_request, response) => {
   response.sendFile('index.html', { root: 'dist/app' });
 });
 
-app.listen(port, () => {
-  console.log(`Server listening on port ${port}!`);
+connectDatabase(process.env.VITE_MONGODB_URI || '').then(() => {
+  app.listen(port, () => {
+    console.log(`Server listening on port ${port}!`);
```
Comment (janhendrikstruve) [inline src/server.ts:40]: I guess this is a console.log, that should stay? :)

**Review by lajanzen — APPROVED** [review]

**Review by janhendrikstruve — APPROVED** [review]
Great Code! Just found some console.logs, so maybe you should have a look, if the should stay or not :) (if they should stay, i would give the first three ones a more specific name)

**Pull request merged**